test/autotest.py

```diff
@@ -741,7 +741,7 @@ def saveResultsNMI():
 runTest("dmtcp5",        2, ["./test/dmtcp5"])
 resource.setrlimit(resource.RLIMIT_STACK, oldLimit)
 
-# Test for a bunch of system calls. We want to use the 'xc' mode for
+# Test for a bunch of system calls. We want to use the 'bc' mode for
 # checkpointing so that the process is killed right after checkpoint. Otherwise
```

Comment:
> so that the process is killed right after checkpoint"

Could you please update the comment? This statement is no longer valid.

Reply:
Please see my comment below. I thought 'bc' is for "blocking until done".

Reply:
When I have time, I'll look mroe closely.  Apparently in the code, 'xc' and 'bc' were always synonyms.  I think that it causes exit after first checkpoint.  But I'd need to check that to be sure.

Also, what exactly does 'blocking until done' mean?  This is not a meaningful phrase for me.  Should we update teh English explanation of the command?

Reply:
>  Should we update teh English explanation of the command?

Yes.

I think 'xc' is to kill the program after resuming from a checkpoint, and 'bc' is for forcing 'dmtcp_command' to block or wait synchronously until the checkpointing is completed. With just 'c', `dmtcp_command` sends the command to the coordiantor and returns immediately, without waiting for the checkpoint to complete. I have used both the commands in the past for different use cases. I don't know when 'xc' was deprecated.

Reply:
@rohgarg, You're correct in all of the above.  I'm now using 'xc' again as I should.  Also, if you look at commit ee798b0 , you'll see why we had commented out the documentation on the 'xc' command.  That was a mistake and it is now documented as it should be.

Reply:
And since this PR has been waiting for so long, we've already changed 'xc' to 'Kc' in the above code.  Wherever 'xc' is written, the code now uses 'Kc', after doing a rebase and resolving conflicts.